Vic2ToHoI4/Source/V2World/Pops/Pop.h

```diff
@@ -12,29 +12,43 @@ namespace Vic2
 {
 
 
+struct PopOptions
+{
+	std::string type;
+	std::string culture = "no_culture";
+	int size = 0;
+	double literacy = 0.0;
+	double militancy = 0.0;
+	std::map<std::string, float> popIssues;
+};
+
+
 class Pop
 {
   public:
-	class Builder;
-	class Factory;
 	Pop() = default;
-
-	[[nodiscard]] const auto& getType() const { return type; }
-	[[nodiscard]] const auto& getCulture() const { return culture; }
-	[[nodiscard]] const auto& getSize() const { return size; }
-	[[nodiscard]] const auto& getLiteracy() const { return literacy; }
-	[[nodiscard]] const auto& getMilitancy() const { return militancy; }
+	explicit Pop(const PopOptions& options):
+		 type_(options.type), culture_(options.culture), size_(options.size), literacy_(options.literacy),
+		 militancy_(options.militancy), popIssues_(options.popIssues)
+	{
+	}
+
+	[[nodiscard]] const auto& getType() const { return type_; }
+	[[nodiscard]] const auto& getCulture() const { return culture_; }
+	[[nodiscard]] const auto& getSize() const { return size_; }
+	[[nodiscard]] const auto& getLiteracy() const { return literacy_; }
+	[[nodiscard]] const auto& getMilitancy() const { return militancy_; }
 
 	[[nodiscard]] float getIssueSupport(const std::string& issueName) const;
 
   private:
-	std::string type;
-	std::string culture = "no_culture";
-	int size = 0;
-	double literacy = 0.0;
-	double militancy = 0.0;
+	std::string type_;
+	std::string culture_ = "no_culture";
+	int size_ = 0;
+	double literacy_ = 0.0;
+	double militancy_ = 0.0;
```

Comment:
Why the underscores?

Reply:
1. To not conflict with parameter names in member initialization lists in constructors.
2. To make it easier to recognize member variable names inside classes.
3. (minor) To be closer to my employer's style so I don't have to keep switching styles.